src/python/WMComponent/AgentStatusWatcher/ResourceControlUpdater.py

```diff
@@ -99,14 +99,14 @@ def algorithm(self, parameters):
                           (str(self.pendingSlotsSitePercent), str(self.pendingSlotsTaskPercent))) 
             
             if self.queueParams.get('DrainMode', False):
-                agentsNum = 1
+                agentsNum = 5
```

Comment:
If it's in DrainMode, it's better not to split the thresholds among more agents.
It still makes sense to set it do one in here.